src/tigerasi/tiger_controller.py

```diff
@@ -897,15 +912,26 @@ def start_array_scan(self, wait: bool = True):
         self._set_cmd_args_and_kwds(Cmds.ARRAY, card_address=card_address,
                                     wait=wait)
 
-    def reset_ring_buffer(self, wait: bool = True):
-        """Clear the ring buffer contents."""
-        self._clear_ring_buffer(wait=wait)
-        self._rb_axes = []
+    def reset_ring_buffer(self, axis: str = None, wait: bool = True):
+        """Clear the ring buffer contents.
+        See `RING BUFFER MODULE <https://asiimaging.com/docs/ring_buffer>`
```

Comment:
This needs an underscore at the end of it. (Gah! Restructured Text syntax is wonky.)
Like this:
```
`RING BUFFER MODULE <https://asiimaging.com/docs/ring_buffer>`_
```